app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
-  belongs_to :userId
-  belongs_to :postId
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_count
+    update(likes_count: likes.count)
+  end
```

Comment:
- [ ] Please use ` post.increment!(:likes_count)` instead of  ` update(likes_count: likes.count)` so kindly fix this issue :slightly_smiling_face: 
- [ ] Please add `after_save :update_likes_count` before `def update_likes_count` method